lessons/diffusion.yml

```diff
@@ -96,12 +83,15 @@ instructor_notes: |
   - Data formatting issues due to either not pasting in the full contents of the data dump, resetting stats during data collection, etc.
 
   Possibilities to "level up" this activity for more advanced students:
-  - Connect to 2D Lennard-Jones phase diagram (conditions here are in the two-phase solid-liquid region!) and thinking about why solid ends up in middle of box and liquid along the edges (see e.g. https://www.researchgate.net/publication/49750821/figure/fig1/AS:667216336936969@1536088179352/Phase-diagram-of-2D-Lennard-Jones-solid-as-given-in-9-The-first-order-boundaries-are.png) 
+  - Connect to 2D Lennard-Jones phase diagram (conditions here are in the two-phase solid-liquid region!) and thinking about why solid ends up in middle of box and liquid along the edges (see e.g. [here](https://www.researchgate.net/publication/49750821/figure/fig1/AS:667216336936969@1536088179352/Phase-diagram-of-2D-Lennard-Jones-solid-as-given-in-9-The-first-order-boundaries-are.png))
   - "Skeletonize" some of the code cells and ask students to fill in some of the analysis or visualization themselves
   - Get more in-depth into what's going on with the MD, understand why temperature drifts, etc.
   - Have them repeat the second portion at another temperature and think about how/why the results change
   - Dive more into some of the PDE stuff at the end
 
+# Related lessons that complement this one
+related_modules:
```

Comment:
Remove this related_modules keyword since there aren't any. Unfortunately at the moment, having it present but empty will make the build fail.